src/rust-2018/edition-changes.md

```diff
@@ -0,0 +1,47 @@
+# 2018-Specific Changes
+
+The following is a summary of changes that only apply to code compiled with
+the 2018 edition compared to the 2015 edition.
+
+## Rust
+- [Non-lexical lifetimes].
+- [At most once] `?` macro repetition operator.
+- [Path changes]:
+    - `use` declarations must begin with:
+        - `crate` – refers to the current crate.
+        - `self` – refers to the current module.
+        - `super` – refers to the parent module.
+        - An external crate name.
+        - `::` – must be followed by an external crate name. This is required
+          if an external crate has the same name as an in-scope item, to catch
+          possible ambiguities with future module changes.
+    - Paths in `pub(in path)` visibility modifiers must start with `crate`,
+      `self`, or `super`.
+- [Anonymous trait function parameters] are not allowed.
+    - [Trait function parameters] may use any irrefutable pattern when the
+      function has a body.
+- [`dyn`] is a [strict keyword], in 2015 it is a [weak keyword].
+- `async`, `await`, and `try` are [reserved keywords].
```

Comment:
`await` is not reserved

Reply:
```rust
error[E0721]: `await` is a keyword in the 2018 edition
 --> src/lib.rs:1:4
  |
1 | fn await() {}
  |    ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
```

You have to use `#![feature(await_macro)]` to make it unreserved.

Reply:
Oh god.
They future-proofed it with a hack in a lint pass.

Ok, this makes it kinda reserved in practice.